### PR TITLE
Add support of vrrp_notify_fifo*

### PR DESCRIPTION
**Pull request by ymartin-ovh**
#### Pull Request (PR) description
Add support of vrrp_notify_fifo and vrrp_notify_fifo_script

### CODE REVIEW

File: manifests/global_defs.pp [hunk 1]
```diff
@@ -121,6 +125,8 @@
   Optional[Integer[2, 3]] $vrrp_version              = undef,
   Optional[Integer[-1, 99]] $max_auto_priority       = undef,
   Boolean $dynamic_interfaces                        = false,
+  Optional[String] $vrrp_notify_fifo                 = undef,
+  Optional[String] $vrrp_notify_fifo_script          = undef,
```
Comment (bastelfreak) [inline manifests/global_defs.pp:129]: is that always an absolute path? then we could do:
```suggestion
  Optional[Stdlib::Absolutepath] $vrrp_notify_fifo_script = undef,
```

File: manifests/global_defs.pp [hunk 2]
```diff
@@ -121,6 +125,8 @@
   Optional[Integer[2, 3]] $vrrp_version              = undef,
   Optional[Integer[-1, 99]] $max_auto_priority       = undef,
   Boolean $dynamic_interfaces                        = false,
+  Optional[String] $vrrp_notify_fifo                 = undef,
```
Comment (bastelfreak) [inline manifests/global_defs.pp:128]: we could probably enforce an absolute path here as well?

Comment (ymartin-ovh) [reply manifests/global_defs.pp:128]: Yep, I will do that.

**Comment by ymartin-ovh** [comment]
I think centos6 check crash during worker boot time.

**Comment by bastelfreak** [comment]
restarted it and it passed

**Pull request merged**